iceoryx_posh/test/moduletests/test_mepoo_mempool.cpp

```diff
@@ -44,24 +46,63 @@ class alignas(32) MemPool_test : public Test
     iox::mepoo::MemPool sut;
 };
 
-TEST_F(MemPool_test, CTor)
+TEST_F(MemPool_test, MempoolCtorInitialisesTheObjectWithValuesPassedToTheCtor)
 {
+    char memory[8192];
+    iox::posix::Allocator allocator1{memory, 8192U};
+
+    iox::mepoo::MemPool sut1(ChunkSize, NumberOfChunks, &allocator1, &allocator1);
+
+    EXPECT_THAT(sut1.getChunkSize(), Eq(ChunkSize));
+    EXPECT_THAT(sut1.getChunkCount(), Eq(NumberOfChunks));
+    EXPECT_THAT(sut1.getMinFree(), Eq(NumberOfChunks));
 }
 
-TEST_F(MemPool_test, WriteChunks)
+TEST_F(MemPool_test, MempoolCtorWhenChunkSizeIsNotAMultipleOfAlignmentReturnError)
 {
-    uint8_t* chunk = reinterpret_cast<uint8_t*>(sut.getChunk());
-    uint8_t* chunk2 = reinterpret_cast<uint8_t*>(sut.getChunk());
+    char memory[8192U];
+    iox::posix::Allocator allocator1{memory, 100U};
+    uint32_t NotAlignedChunkSize = 33U;
+
+    iox::cxx::optional<iox::Error> detectedError;
+    auto errorHandlerGuard = iox::ErrorHandler::SetTemporaryErrorHandler(
+        [&detectedError](const iox::Error error, const std::function<void()>, const iox::ErrorLevel errorLevel) {
+            detectedError.emplace(error);
+            EXPECT_THAT(errorLevel, Eq(iox::ErrorLevel::FATAL));
+        });
+
+    iox::mepoo::MemPool sut1(NotAlignedChunkSize, NumberOfChunks, &allocator1, &allocator1);
+
+    ASSERT_TRUE(detectedError.has_value());
+    EXPECT_THAT(
+        detectedError.value(),
+        Eq(iox::Error::
+               kMEPOO__MEMPOOL_CHUNKSIZE_MUST_BE_LARGER_THAN_SHARED_MEMORY_ALIGNMENT_AND_MULTIPLE_OF_ALIGNMENT));
+}
 
-    *chunk = 123;
-    *chunk2 = 45;
+TEST_F(MemPool_test, MempoolCtorWhenChunkSizeIsSmallerThan32BytesGetsTerminated)
```

Comment:
Actually, the chunk size has to be greater than or equal to ``MEMORY_ALIGNMENT``. Please make the test name more precise and to make it more robust against changes you could write something like
`` constexpr uint32_t CHUNK_SIZE_SMALLER_THAN_MEMORY_ALIGNMENT = iox::mepoo::MemPool::MEMORY_ALIGNMENT - 1;``